modules/Behavior.js

```diff
@@ -49,10 +53,18 @@
     mixin: {},
 
     /**
+     * The behavior's prepare result will be combined with the view's prepare with the behavior's alias as the namespace.
+     * effectively: { <behaviorName>: behavior.prepare() } will be combined with the view's prepare result.
+     * @method prepare
+     * @return a prepare context suitable to being added to the view's prepare result.
+     */
+    prepare: _.noop,
```

Comment:
grr... i wish that prepare was part of the lifecycle events list. now it's harder to find.